### PR TITLE
finish working through datasources

### PR DESCRIPTION
**Pull request by dr-wesley**
all markdown files in 'datasources' have links to APIs and updates to arguments and attributes. Latter for accuracy and consistency

### CODE REVIEW

File: docs/ExamplesContents.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Terraform Provider Documentation: Examples
+This file lists all the examples in each category.
+
+## Clusters
```
Comment (briangustafson) [inline docs/ExamplesContents.md]: There's really only a single example in Clusters, and that's at (https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb. The files in that directory are all supporting that one example.

File: docs/ExamplesContents.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- [env-vars](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/env-vars)
+- [network](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/network.tf)
+- [outputs](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/outputs.tf)
+- [provider](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/provider.tf)
+- [README](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/README.md)
+- [variables](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples/clusters/Mongodb/variables.tf)
+
+
+## Compute
+- [console_history](https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/resources/core/console_history.md)
```
Comment (briangustafson) [inline docs/ExamplesContents.md]: I would expect this index to only point at examples under https://github.com/oracle/terraform-provider-baremetal/tree/master/docs/examples.

File: docs/datasources/core/ipsec_connection_device_status.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # oci\_core\_ipsec\_status
 
-Gets a list of internet gateways.
+**API:** [IPSecConnectionDeviceStatus Reference][6c32ac75]
+
+  [6c32ac75]: https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/IPSecConnectionDeviceStatus/ "IPSecConnectionDeviceStatusReference"
+
+~~Gets a list of internet gateways.~~ Gets information about the status of the IPSec connection.
```
Comment (briangustafson) [inline docs/datasources/core/ipsec_connection_device_status.md]: Should "Gets a list of internet gateways." be removed?

Comment (dr-wesley) [reply docs/datasources/core/ipsec_connection_device_status.md]: I think so, it's not accurate

**Comment by dr-wesley** [comment]
cleaned this up so it should be good to go.

**Comment by dr-wesley** [comment]
Just need a quick look and approval and this should be good to go

File: docs/datasources/core/volumes.md [hunk 4]
```diff
@@ -27,11 +31,10 @@ The following attributes are exported:
...
-* `display_name` - A user-friendly name. Does not have to be unique.
-* `id` - The OCID of the Volume.
-* `state` - The current state of the volume. [PROVISIONING,RESTORING,AVAILABLE,TERMINATING,TERMINATED,FAULTY]
-* `size_in_mbs` - (Deprecated) The size of the volume, in MBs.
-* `size_in_gbs` - The size of the volume, in GBs.
-* `time_created` - The date and time the Volume was created.
+* `display_name` - A user-friendly name. Does not have to be unique, and it's changeable. Avoid entering confidential information.
+* `id` - The OCID of the volume.
+* `state` - The current state of the volume. Allowed values are: [PROVISIONING, RESTORING, AVAILABLE, TERMINATING, TERMINATED, FAULTY]
+* `size_in_mbs` - The size of the volume, in MBs. The size must be a multiple of 1024.
```
Comment (codycushing) [inline docs/datasources/core/volumes.md]: Looks like we're dropping the new `size_in_gbs` entry here

File: docs/datasources/core/volume_backups.md [hunk 5]
```diff
@@ -28,13 +32,11 @@ The following attributes are exported:
...
-* `size_in_gbs` - The size of the volume, in GBs.
-* `time_created` - The date and time the Volume was created.
-* `time_requested` - The date and time the request to create the volume backup was received.
-* `unique_size_in_mbs` - (Deprecated) The size used by the backup, in MBs. It is typically smaller than sizeInMBs, depending on the space consumed on the volume and whether the backup is full or incremental.
-* `unique_size_in_gbs` - The size used by the backup, in GBs. It is typically smaller than sizeInGBs, depending on the space consumed on the volume and whether the backup is full or incremental.
-* `volume_id` - The OCID of the Volume.
+* `display_name` - A user-friendly name. Does not have to be unique, and it's changeable. Avoid entering confidential information.
+* `id` - The OCID of the volume backup.
+* `state` - The current state of the volume. Allowed values are: [CREATING, AVAILABLE, TERMINATING, TERMINATED, FAULTY, REQUEST_RECEIVED]
+* `size_in_mbs` - The size of the volume, in MBs. The value must be a multiple of 1024.
```
Comment (codycushing) [inline docs/datasources/core/volume_backups.md]: Restore `size_in_gbs` here

File: docs/datasources/core/volume_backups.md [hunk 6]
```diff
@@ -28,13 +32,11 @@ The following attributes are exported:
...
+* `time_created` - The date and time the volume was created, in the format defined by RFC3339.  Example: `2016-08-25T21:10:29.600Z`.
+* `time_requested` - The date and time the request to create the volume backup was received, in the format defined by RFC3339.
+* `unique_size_in_mbs` - The size used by the backup, in MBs. It is typically smaller than sizeInMBs, depending on the space consumed on the volume and whether the backup is full or incremental.
```
Comment (codycushing) [inline docs/datasources/core/volume_backups.md]: Restore `unique_size_in_gbs` here

File: docs/datasources/core/volumes.md (hunk 4 above)
Comment (codycushing) [inline docs/datasources/core/volumes.md]: Restore `size_in_gbs` here

**Comment by dr-wesley** [comment]
dan>> I have absolutely no idea what 'squash' this commit is... however, fixes are in so I'll punt this to you.

**Pull request merged**